src/All/README.md

```diff
@@ -40,11 +40,15 @@ allGood([ 'nice', '00', null ])
 All.empty :: () -> All
 ```
 
+<<<<<<< HEAD
```

Comment:
Looks like there was some rebasing silliness

Reply:
HAH! It scared my pants off.

Reply:
Do I manually fix these and merge?